deploy/examples/operator-openshift.yaml

```diff
@@ -584,6 +592,17 @@ data:
   # Defaults to all the labels mentioned in
   # https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/#osd-topology
   # CSI_CRUSH_LOCATION_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
+
+  # Specify below field to instruct Kubernetes to skip any attach operation altogether
```

Comment:
Could you also make the comment update here and in operator.yaml?

Reply:
done